README.md

```diff
@@ -30,6 +30,12 @@ type: `boolean`
 Declare whether your user is in the USA or not. Required – *throws an error if
 it's missing.*
 
+#### options.pubData
```

Comment:
Would be nice to show how to use this object by updating the example given in line 42 to include how to pass the browserId. Doesn't need to be part of this PR though.